### PR TITLE
UserLogins#Delete (aka logout)

### PR DESCRIPTION
**Pull request by wintondeshong**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: dotnet/api/Presentation/Web/Controllers/Api/V1/ApiController.cs [hunk 1]
```diff
@@ -26,5 +27,16 @@ public class ApiController : AndcultureCode.CSharp.Web.Controllers.ApiController
         public ApiController(IStringLocalizer localizer) : base(localizer) { }
 
         #endregion Constructors
+
+        #region Public Methods
+
+        /// <summary>
+        /// Responds with HTTP 200 Ok
+        /// TODO: Abstract to AndcultureCode.CSharp.Web.Controllers.Controller
```
Comment (wintondeshong) [inline dotnet/api/Presentation/Web/Controllers/Api/V1/ApiController.cs:35]: Waiting on PR https://github.com/AndcultureCode/AndcultureCode.CSharp.Web/pull/8 and I'll remove this change before merging.

File: dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/Api/V1/UserLogins/UserLoginsControllerTest.cs [hunk 2]
```diff
@@ -35,6 +36,46 @@ ITestOutputHelper output
...
+            var requestedUserLogin = Create<UserLogin>(e => e.UserId = user.Id);
+
+            MockAuthenticatedUser(user, currentUserLogin);
+
+            // Act & Assert
+            Sut.Delete(requestedUserLogin.Id).Result.AsUnauthorized<UserLoginDto>();
+        }
+
+        [Fact]
+        public void Delete_When_Logout_Successful_Then_Deletes_Login_Adn_Returns_Ok()
```
Comment (brandongregoryscott) [inline dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/Api/V1/UserLogins/UserLoginsControllerTest.cs:58]: Typo: `Adn`

File: dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/ControllerTest.cs [hunk 3]
```diff
@@ -352,6 +365,26 @@ protected void SetSutClaimsPrincipal(User user)
...
+        protected void SetSutClaimsPrincipal(User user, UserLogin userLogin)
+        {
+            if (Sut == null)
+            {
+                throw new InvalidOperationException("[ControllerTest] Sut MUST be set");
+            }
+
+            Sut.ApiClaimsPrincipal = new ApiClaimsPrincipal
+            {
+                RoleId = userLogin.RoleId,
```
Comment (brandongregoryscott) [inline dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/ControllerTest.cs:382]: Should we null check `userLogin` before accessing properties off of it? Based on `MockAuthenticatedUser` which calls this setup method, it's not required

Comment (wintondeshong) [reply dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/ControllerTest.cs:382]: Yeah, wouldn't hurt!

File: dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/Api/V1/UserLogins/UserLoginsControllerTest.cs (hunk 2 above)
Comment (wintondeshong) [reply dotnet/api/Tests/Presentation/Web.Tests/Integration/Controllers/Api/V1/UserLogins/UserLoginsControllerTest.cs:58]: Bah, good catch

**Pull request merged**